pandas/core/series.py

```diff
@@ -1002,36 +1002,86 @@ def _set_value(self, label, value, takeable=False):
 
     def reset_index(self, level=None, drop=False, name=None, inplace=False):
         """
-        Analogous to the :meth:`pandas.DataFrame.reset_index` function, see
-        docstring there.
+        Reset the index of the Series.
```

Comment:
so this is not correct, ``.reset_index(drop=False)`` (the default) on a Series will return a DataFrame. The columns will be the name of the Series and a new column of the index, with its name being the name of the column or 'index' if its not named.